api/envoy/api/v2/eds.proto

```diff
@@ -89,7 +89,12 @@ message ClusterLoadAssignment {
     // multiplied by the overprovisioning factor drops below 100.
     // With the default value 140(1.4), Envoy doesn't consider a priority level
     // or a locality unhealthy until their percentage of healthy hosts drops
-    // below 72%.
+    // below 72%. For example:
+    //
+    // .. code-blocK:: json
```

Comment:
typo: "blocK" (maybe it's case-insensitive but might want to check rendered docs or just fix to lower case)